extensions/wikia/EditPageLayout/css/core/wide.scss

```diff
@@ -263,6 +241,35 @@
 	}
 }
 
+#EditPage {
+	&.toolbar-expanded {
+		.preview_box {
+			border: 0;
+			clear: both;
+
+			h3 {
+				margin-top: 8px;
+			}
+
+			.preview_mobile {
+				padding-top: 3px;
+			}
+
+			.preview_desktop {
+				padding-top: 5px;
+			}
+		}
+
+		#wpDiff.diff-link {
```

Comment:
Yep, that's tricky but ```toolbar-expanded``` class is toggle via javascript after screen resize so we need to load these css definitions.